src/app/tab/tab.component.html

```diff
@@ -32,6 +32,14 @@
           </tab-time>
         </ng-container>
       </ng-container>
+      <ng-container *ngIf="settings.config.covidData.enabled && settings.config.covidData.countries.length > 0">
+        <ng-container *ngFor="let c of settings.config.covidData.countries; let i = index;" >
+          <tab-covid *ngIf="c.position === 'nw'" [index]="i" [countryCode]="c.code"
+          [style.order]="shared.getOrder(c.id)"
+          [style.fontSize]="shared.getFontSize(c.textScaling, 20)"
+          [style.transform]="shared.getOffset(c.offset)"></tab-covid>
+        </ng-container>
```

Comment:
You will need to add a margin option, that is what you will set on the container here, and it will adjust the spacing between the covid data and other elements around it.:

```
[style.margin]="this.shared.getMargin(this.settings.config.covidData.countries[index].margin)"
```